schema/experiment.yml

```diff
@@ -87,10 +87,11 @@ $defs:
   ControlVariant:
     type: object
     additionalProperties: false
+    description: A variant used as an experimental control to measure assay concorance
     properties:
-      vrs:
+      allele:
```

Comment:
changed this to allele for clarity–while it is a VRS object, this field is constrained to Allele, and it is better to be as precise as possible with field names when it comes to describing the field value.